v4/export/sql.go

```diff
@@ -570,25 +570,98 @@ func buildSelectField(db *sql.Conn, dbName, tableName string, completeInsert boo
 	return "*", len(availableFields), nil
 }
 
-func buildWhereClauses(handleColNames, handleVals []string) []string {
-	if len(handleVals) == 0 {
+func buildWhereClauses(handleColNames []string, handleVals [][]string) []string {
```

Comment:
please add a unit test for this function.

Reply:
added in https://github.com/pingcap/dumpling/pull/260/commits/3959b53251a2c2afdb0cc0ad5b46d1fe69244789